contributing/publishing.md

```diff
@@ -72,6 +72,23 @@ After the pre-publish steps have run and main has been rebased off of the releas
 
 ---
 
+# Publishing a Beta Pre-Release
+
+If there is a release with high-risk or large-scale changes, we can publish a pre-release to allow advanced users to test early. VSIX artifacts are uploaded to a github release as with our usual release but there is no publish to NPM or the VS Code Marketplace (yet).
+
+## Steps
+
+1. Create a release branch, and increment the version using Lerna, as shown in the `create-release-branch.js` file, starting at the creation of the release branch.
```

Comment:
nit: should we spell this out here?   That script may not live forever so this doc could get out of date fairly quickly :)

Reply:
I'd like to keep it for now just in case some other sUpEr urgent work comes up and keeps this relevant; but keep me honest to updating it once subsequent PRs make this out of date!

Reply:
we're trying to get most of our release tasks into the gh ui.  for example, we have a CLI command and an action that creates our release PR.  

https://github.com/salesforcecli/sfdx-cli/blob/sm/gha/.github/workflows/makePRforRC.yml